tests/test_unit/conftest.py

```diff
@@ -300,6 +303,13 @@ async def create_connection_auth_data(request) -> dict | None:
     return None
 
 
+@pytest_asyncio.fixture
+async def connection_type(request) -> str | None:
```

Comment:
How do we use this fixture?

Reply:
For example, in `group_connection` fixture. Without this definition tests fail with `fixture doesn't exist`. There are also similar fixtures in this file

Reply:
I see, this is just the same as `create_connection_data` and `create_transfer_data`

Reply:
Do you mean we should combine these fixtures? I guess `connection_type` is not a part of them

Reply:
No, I just wandered why this fixture just returns the original param as is, but it is used in the fixture chain which is configured via parametrize+indirect. Not ideal, but for now it's okay.